frontend/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": false,
+  "useTabs": true
```

Comment:
Veldig fan av at vi bruker prettier, veldig lite fan av `useTabs` 😬

Reply:
Mest fordi jeg prøvde å få ned diff mellom min commit og setupet fra Kathrine. Uten at jeg fikk det helt til. Mulig det er pga eslint. Jeg pleier selv egentlig å bruke spaces, ikke tabs, så jeg har ingenting imot å ta den bort igjen (den defaulter til false uansett).

Reply:
Greide å oppnå det jeg ville med å sette den til false + sette tabWidth. Nå kan du være fan igjen.

Reply:
Jaa joo, det funker som  det skal nå, men jeg skjønner vel fremdeles ikke hvorfor vi må sette en haug med default options.

Alle optionene som er satt i `.prettierrc` er nå det samme som default 🤔 
Er det fordi du har en `.editorconfig`?

Reply:
Antageligvis, jeg har iallefall disse (bortsett fra useTabs, dog) satt te ikke default, med GitHub profile sync. Så dette er for å override mine andre settings.

Reply:
Sagt på en annen måte; uten disse får du mine commits med bl.a. single quotes.